dev-java/eclipse-ecj/eclipse-ecj-4.33.ebuild

```diff
@@ -22,16 +22,16 @@ SLOT="4.33"
 KEYWORDS="~amd64 ~arm64 ~ppc64"
 
 BDEPEND="app-arch/unzip"
-COMMON_DEP="app-eselect/eselect-java"
```

Comment:
the dependency is not useless as we have `eselect ecj`:
```
$ eselect ecj
Usage: eselect ecj <action> <options>

Standard actions:
  help                      Display help text
  usage                     Display usage information
  version                   Display version information

Extra actions:
  list                      List available ECJ targets
  set <target>              Set a new ECJ target
    target                    Target name or number (from 'list' action)
  show                      Show current ECJ target
  update <target>           Set the ECJ target to the latest if the current target is invalid or if the given target is the latest
    target (optional)         Target name (from 'list' action)
```

Reply:
Should I put `optfeature` (or what was its name?)?

Reply:
i'd simply keep the dependency, it just can be moved to `RDEPEND`.

Reply:
> i'd simply keep the dependency, it just can be moved to `RDEPEND`.

But it is definitely not needed, just optional, for running `eclipse-ecj`. Isn't this a typical use case for `optfeature`?

Reply:
well, it's the same case like for `maven-bin`. you use `eselect` to pick the version that is used from command line. `maven-bin` ebuild just goes further and also updates the selected implementation which we don't do for `ecj`, though we might or maybe even should.

Reply:
> well, it's the same case like for `maven-bin`. you use `eselect` to pick the version that is used from command line. `maven-bin` ebuild just goes further and also updates the selected implementation which we don't do for `ecj`, though we might or maybe even should.

@fordfrog I will close this PR leaving the `eclipse-ecj-4.33` ebuild as is. It really doesn't hurt.